assets/app/scripts/app.js

```diff
@@ -194,6 +195,14 @@ angular
         templateUrl: 'views/browse/service.html',
         controller: 'ServiceController'
       })
+      .when('/project/:project/browse/storage', {
+        templateUrl: 'views/storage.html',
+        controller: 'StorageController'
+      })
+      .when('/project/:project/browse/persistentvolumeclaims/:pvc', {
```

Comment:
Concerned with that pattern we might run into conflicts with action names and resource names, even if it reads better.